### DISCUSSION TITLE
JCal support

### DISCUSSION THREAD

**Discussion by abe-101** [collective/icalendar#702] [Q&A]
I think it would be interesting to support the JCal format defined in [rfc7265](https://datatracker.ietf.org/doc/html/rfc7265)
both as a way to import from jcal format as well as to export into jcal format

issue opened: https://github.com/collective/icalendar/issues/703

**Comment by niccokunzmann** [comment]
Hi,
I quite like the idea.
This makes it also easier for some applications to convert and access the .ics files: If we can provide a way to convert between both.

**Reply by abe-101** [reply]
Yes
 [ical.js](https://github.com/kewisch/ical.js/) does this
 Thats what gave me the idea

**Reply by niccokunzmann** [reply]
And I remember: For any web application e.g. with flask/Django that provides calendar output as JSON, with jCal, there is then an easy way to dump all the data! Dates and parameters usually require more programming. jCal would make development and debugging much more accessible for all small projects! :heart:

**Comment by abe-101** [comment]
My use case would be to add a feature to [MergeCal.org](https://mergecal.org) 
which allow users to export the feed as json

can also be handy with tools like [pipedream](https://pipedream.com) where you can only pass json btw steps

**Comment by niccokunzmann** [comment]
jCal is added now: https://icalendar.readthedocs.io/en/latest/how-to/jcal.html
